android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/webcontent/display/WebContentWithCustomCssDisplayView.java

```diff
@@ -0,0 +1,35 @@
+package com.liferay.mobile.screens.viewsets.defaultviews.webcontent.display;
+
+import android.content.Context;
+import android.util.AttributeSet;
+
+/**
+ * @author Sarai Díaz García
+ */
+
+public class WebContentWithCustomCssDisplayView extends WebContentDisplayView {
```

Comment:
Why are you creating a new view? Why not including this CSS as the default value in the parent view?